cdap-sentry/cdap-sentry-extension/README.rst

```diff
@@ -60,73 +58,84 @@ To skip tests, execute::
-       also download this file from the Actions -> Download Client Configuration drop down link in the "Configuration"
-       tab of the Sentry Service, and copy it to a location of your choice on the CDAP Master Host.
-     - -
-   * - ``security.authorization.extension.config.sentry.admin.group``
-     - A unix group configured as an admin group in the Sentry Service (identified by ``sentry.service.admin.group``
-       in the ``sentry-site.xml`` used by the Sentry Service). This group is used while granting ``ALL`` privileges
-       to a user when he/she successfully creates an entity, as well as revoking privileges when an entity is deleted.
-       It is also required to list privileges and roles in Sentry for enforcing authorization on CDAP entities.
-       It is recommended that the ``cdap`` user (which runs the CDAP Master) be added to the
-       ``sentry.service.admin.group`` configuration, but any other user is also acceptable.
-     - ``cdap``
-   * - ``security.authorization.extension.config.instance.name``
-     - String to use to identify the CDAP Instance.
-     - ``cdap``
-
-- Restart CDAP Master.
+The CDAP Master, which is also a client for the Sentry service, requires the CDAP Sentry
+Binding classes as well as its dependencies. To deploy the ``cdap-sentry`` authorization
+extension:
+
+- Install the ``cdap-sentry-extension/cdap-sentry-binding/target/cdap-sentry-binding-*.jar`` 
+  at a known location on your CDAP Master host
+- Set these properties in the ``cdap-site.xml`` that the CDAP Master uses:
+
+  .. list-table::
+     :widths: 20 70 10
+     :header-rows: 1
+
+     * - Parameter
+       - Description
+       - Default Value
+     * - ``security.authorization.extension.jar.path``
+       - The absolute path, including the filename, of the ``cdap-sentry-binding-*.jar``
+         file on the local file system of the CDAP Master
+       - *none*
+     * - ``security.authorization.extension.config.sentry.site.url``
+       - The absolute path of the client-side ``sentry-site.xml`` on the local file system
+         of the CDAP Master. Note that if Apache Sentry is managed via Cloudera Manager, you can
+         add a Sentry Gateway role to the CDAP Master host, and the file will be available
+         at ``/etc/sentry/conf/sentry-site.xml`` on the CDAP Master host. Alternatively, you
+         can download this file from the *Actions -> Download Client Configuration* drop
+         down link in the "Configuration" tab of the Sentry Service, and copy it to a
+         location of your choice on the CDAP Master host.
+       - *none*
+     * - ``security.authorization.extension.config.sentry.admin.group``
+       - A UNIX group configured as an admin group in the Sentry Service (identified by
+         ``sentry.service.admin.group`` in the ``sentry-site.xml`` used by the Sentry
+         Service). This group is used when granting ``ALL`` privileges to a user when they
```

Comment:
forgot to change this. We don't have ``ALL`` as an alias anymore. Can you just make it non-literal (just 'all' - no formatting)?